From c4016c87db259a5a6f0b15ae19760f9f881b6b77 Mon Sep 17 00:00:00 2001
From: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
Date: Mon, 28 Nov 2022 14:59:16 +0100
Subject: [PATCH 167/529] ARM: configs: enable UCSI and STM32G0 driver in
 multi_v7 addons fragment

Enable Type-C UCSI framework, and STM32G0 UCSI driver as module.

Signed-off-by: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
Change-Id: Iae5e3219e9f5c5cced550a5a8f002f704e828861
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/279506
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 arch/arm/configs/fragment-02-multiv7_addons.config | 2 ++
 1 file changed, 2 insertions(+)

--- a/arch/arm/configs/fragment-02-multiv7_addons.config
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -46,6 +46,8 @@ CONFIG_SRAM=y
 CONFIG_TEE=y
 CONFIG_THERMAL=y
 CONFIG_TMPFS=y
+CONFIG_TYPEC_UCSI=m
+CONFIG_UCSI_STM32G0=m
 CONFIG_USELIB=y
 CONFIG_VFP=y
 CONFIG_ZBOOT_ROM_BSS=0x0
